lmnet/lmnet/metrics/mean_average_precision.py

```diff
@@ -104,10 +104,10 @@ def _streaming_tp_fp_array(
             scores = tf.boolean_mask(scores, mask)
 
         # Local variables accumlating information over batches.
-        tp_value = metrics_impl._create_local('tp_value', shape=[0, ], dtype=tf.bool, validate_shape=False)
-        fp_value = metrics_impl._create_local('fp_value', shape=[0, ], dtype=tf.bool, validate_shape=False)
-        scores_value = metrics_impl._create_local('scores_value', shape=[0, ], validate_shape=False)
-        num_gt_boxes_value = metrics_impl._create_local('num_gt_boxes_value', shape=[], dtype=tf.int64)
+        tp_value = metrics_impl.metric_variable(name='tp_value', shape=[0, ], dtype=tf.bool, validate_shape=False)
```

Comment:
You  changed `_create_local` to `metric_variable` because of TensorFlow 1.12 dose not have `metrics_impl._create_local`.

Are there any other reasons?

Reply:
There is no other reason.

Reply:
ok thanks.